code/__DEFINES/mobs.dm

```diff
@@ -600,6 +600,26 @@
 #define AI_EMOTION_BLUE_GLOW "Blue Glow"
 #define AI_EMOTION_RED_GLOW "Red Glow"
 
+///Defines for AI hologram preferences
+#define AI_HOLOGRAM_DEFAULT "Default"
+#define AI_HOLOGRAM_BEAR "Bear"
+#define AI_HOLOGRAM_CARP "Carp"
+#define AI_HOLOGRAM_CHICKEN "Chicken"
+#define AI_HOLOGRAM_CORGI "Corgi"
+#define AI_HOLOGRAM_COW "Cow"
+#define AI_HOLOGRAM_CRAB "Crab"
+#define AI_HOLOGRAM_FOX "Fox"
+#define AI_HOLOGRAM_GOAT "Goat"
+#define AI_HOLOGRAM_CAT "Cat"
+#define AI_HOLOGRAM_CAT_2 "Cat Alternate"
+#define AI_HOLOGRAM_PARROT "Parrot"
+#define AI_HOLOGRAM_PUG "Pug"
+#define AI_HOLOGRAM_SPIDER "Spider"
+#define AI_HOLOGRAM_FACE "Floating Face"
+#define AI_HOLOGRAM_XENO "Xeno Queen"
+#define AI_HOLOGRAM_NARSIE "Narsie"
+#define AI_HOLOGRAM_RATVAR "Ratvar"
```

Comment:
Can this be ABC ordered?

Reply:
Applied with https://github.com/tgstation/tgstation/pull/75740/commits/7dcb31751461310138eb639eb0fa945d11a5d694